src/main/java/org/openstreetmap/atlas/geography/atlas/statistics/AtlasStatistics.java

```diff
@@ -216,8 +229,11 @@ private AtlasStatistics(final Map<StatisticKey, StatisticValue> data)
     @Override
     public boolean equals(final Object other)
     {
-        return other instanceof AtlasStatistics
-                ? ((AtlasStatistics) other).getData().equals(getData()) : false;
+        if (other instanceof AtlasStatistics)
```

Comment:
This change gets rid of a minor code smell: https://sonarcloud.io/project/issues?id=org.openstreetmap.atlas%3Aatlas&issues=AWSBg5FGl6vpliCeK60O&open=AWSBg5FGl6vpliCeK60O